README.analytics.md

```diff
@@ -0,0 +1,136 @@
+stack in a single instance. ELK = Elasticsearch + Logstash + Kibana. This is the stack of components used
+for processing and indexing MH analytics data, particularly usertracking events.
+
+* Modify your cluster config to include the following layer config:
+
+```
+      {
+        "name": "Analytics",
+        "shortname": "analytics",
+        "enable_auto_healing": true,
+        "install_updates_on_boot": true,
+        "type": "custom",
+        "auto_assign_elastic_ips": true,
+        "auto_assign_public_ips": true,
+        "use_ebs_optimized_instances": true,
+        "custom_recipes": {
+            "setup": [
+                "mh-opsworks-recipes::set-timezone",
+                "mh-opsworks-recipes::fix-raid-mapping",
+                "mh-opsworks-recipes::set-bash-as-default-shell",
+                "mh-opsworks-recipes::install-utils",
+                "mh-opsworks-recipes::enable-postfix-smarthost",
+                "mh-opsworks-recipes::install-custom-metrics",
+                "mh-opsworks-recipes::create-alerts-from-opsworks-metrics",
+                "mh-opsworks-recipes::enable-enhanced-networking",
+                "mh-opsworks-recipes::install-elasticsearch",
+                "mh-opsworks-recipes::install-logstash-kibana",
+                "mh-opsworks-recipes::clean-up-package-cache"
+            ],
+            "configure": [
+                "mh-opsworks-recipes::install-ua-harvester",
+                "mh-opsworks-recipes::configure-logstash-kibana"
+            ],
+            "shutdown": [
+                "mh-opsworks-recipes::remove-alarms"
+            ]
+        },
+        "volume_configurations": [{
+            "mount_point": "/vol/elasticsearch_data",
+            "number_of_disks": 1,
+            "size": "300",
+            "volume_type": "gp2"
+        }],
+        "instances": {
+            "number_of_instances": 1,
+            "instance_type": "m4.large",
+            "root_device_type": "ebs"
+        }
+      }
+```      
+
+* update your "custom_json" block to provide a user/pass combo for http auth:
+
+```
+    {
+        "elk": {
+            "http_auth": {
+                "user": "user",
+                "pass": "pass"
+            }
+        }
+    }
+```
+* For a list of settings and defaults for the `"elk"` custom config, see the
+  `get_elk_info` [recipe helper](https://github.com/harvard-dce/mh-opsworks-recipes/blob/master/libraries/default.rb) method.
+  merged to show all the possible elk settings.
+* for existing clusters you may need to manually update your cluster's instance
+  profile to add SQS access. You can find it by viewing any of your stack's
+  instances in the ec2 console, then find the "IAM Role" in the instance
+  description, click that, then "Edit Policy" and add `"sqs:*"` to the allowed
+  action list.
+* execute `./bin/rake admin:cluster:init` and `./bin/rake stack:instances:start`
+  to initialize and start the new layer/instance.
+
+## Cluster delete clean-up
```

Comment:
I'd probably reorder this to be a bit more chronological - or maybe reorganize to make it clear what you have to do in what order to get started.